frontend/src/plugins/action-context-panel/index.tsx

```diff
@@ -291,7 +291,7 @@ const Construct: FunctionComponent<ConstructProps> = ({
                     return [
                         {
                             name: 'MOVE_MOBILE_UNIT',
-                            args: [mobileUnitKey, q, r, s],
+                            args: [mobileUnitKey, _zone, q, r, s],
```

Comment:
the underscore in `_zone` was to hint to the linter that we know it's an unused var ... since it's used now, we can remove `_`  

```suggestion
                            args: [mobileUnitKey, zone, q, r, s],
```